lat0: -72.0791168212891
lon0: 39.3448753356934
yaw0: 0.0
X0:  -72.0791168212891
Y0:  39.3448753356934
Psi0: 0.050772758725122
V0:   0.0